src/work-allocation-2/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -133,7 +133,7 @@ export class TaskListWrapperComponent implements OnDestroy, OnInit {
     return {
       service: TaskService.IAC,
       defaultSortDirection: SortOrder.ASC,
-      defaultSortFieldName: this.getDateField('dueDate'),
+      defaultSortFieldName: 'priority',
```

Comment:
Same as previous comment about constant variable - ignore if to do with mocks

Reply:
Default sort order is changed from dueDate to priority. Yes it will be handled in mock based on value passed

Reply:
Great